src/Extension/CoreExtension.php

```diff
@@ -1565,11 +1565,11 @@ function twig_include(Environment $env, $context, $template, $variables = [], $w
         if (!$alreadySandboxed = $sandbox->isSandboxed()) {
             $sandbox->enableSandbox();
         }
-    }
 
-    // if a Template instance is passed, it might have been instantiated outside of a sandbox, check security
-    if ($template instanceof TemplateWrapper || $template instanceof Template) {
-        $template->unwrap()->checkSecurity();
+        // if a Template instance is passed, it might have been instantiated outside of a sandbox, check security
+        if ($template instanceof TemplateWrapper || $template instanceof Template) {
+            $template->unwrap()->checkSecurity();
```

Comment:
shouldn't this check also be done when the sandbox is already active, even if the `$sandboxed` argument is not explicitly set to `true` ?

Reply:
In that case, the check has already been done.